lang/es.js

```diff
@@ -79,7 +79,7 @@ var language = {
     "download_py_multiple": "Este proyecto contiene varios archivos que no se guardarán con este formato.\nRecomendamos descargar el archivo Hex, que contiene todo el proyecto y puede volver a cargarse en el editor.\n\n¿Estas seguro que quieres descargar el {{file_name}} archivo solo?"
   },
   "code": {
-    "start": "# Añade tu código Python aquí. Por ejemplo,\n\n \n"
+    "start": "Añade tu código Python aquí. Por ejemplo."
```

Comment:
This should probably end with a comma?

Reply:
Have edited this in crowdin and synced the file. Crowdin complained that the source string ended in a full stop, so i had to override it.